nautobot/core/views/generic.py

```diff
@@ -384,6 +385,10 @@ def get(self, request, *args, **kwargs):
         form = self.model_form(instance=obj, initial=initial_data)
         restrict_form_fields(form, request.user)
 
+        relationships_errors = get_relationships_errors(request, obj)
+        if len(relationships_errors) > 0:
+            return redirect(get_route_for_model(obj, "list"))
```

Comment:
Seems wrong to me - why should we ever get errors on a GET (i.e., loading the "edit" page initially)? And wouldn't this prevent us from ever editing the object as a result?

Reply:
My thinking was that, as the requirement is for editing or creating objects, the user should not be allowed be see the edit page without required relationship objects being present in the database. Is there a better/more UX experience-friendly way?